sdk/management/docs/AUTH.md

```diff
@@ -0,0 +1,100 @@
+# Authentication in Azure Management Library for Java
+
+To use the APIs in the Azure Management Library for Java, as the first step you need to create an authenticated client. This document is to introduce several possible approaches for authentication.
+
+## Prerequisites
+
+Before authenticating the Azure client, there are a few authentication classes you need to know before you decide which kind of authentication would meet your requirement.
+
+* TokenCredential
+  * The `TokenCredential` is an interface in the `azure-core` package for credentials that can provide a token. 
+  * Azure Identity offers multiple implementations of the `TokenCredential` class in the `azure-identity` package. To learn more, see [credentials in Azure Identity](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/identity/azure-identity#credentials).
+
+Sample code to create a simple `ClientSecretCredential`:
+
+```java
+ClientSecretCredential clientSecretCredential = new ClientSecretCredentialBuilder()
+    .clientId("<YOUR_CLIENT_ID>")
+    .clientSecret("<YOUR_CLIENT_SECRET>")
+    .tenantId("<YOUR_TENANT_ID>")
+	.authorityHost("<AZURE_AUTHORITY_HOST>")
+    .build();
+```
+
+Please note, most of credentials require a [service principal registration](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal). Alternatively, you could also register a service principal in [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli) or [Azure Powershell](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-authenticate-service-principal-powershell).
+
+* AzureProfile
+  * The `AzureProfile` is a class holding `AzureEnvironment`, `subscriptionId`, `tenantId` to configure the requests sending to wire. 
+  * The `subscriptionId` is mandantory for most resource management while the `tenantId` would be required only for Graph RBAC. They can be set via environment variables.
+  
+|variable name|value
+|-|-
+|`AZURE_TENANT_ID`|id of the principal's Azure Active Directory tenant
+|`AZURE_SUBSCRIPTION_ID`|id of the subscription for the Azure resources
+
+Sample code to create a `AzureProfile`:
+
+```java
+//AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE, true);
+AzureProfile profile = new AzureProfile("<YOUR_TENANT_ID>", "<YOUR_SUBSCRIPTION_ID>", AzureEnvironment.AZURE);
+```
+
+* HttpPipelinePolicy
+  * The `HttpPipelinePolicy` is an interface that process provided request context and invokes the next policy. To learn more, see [policy in Azure Core](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/core/azure-core/src/main/java/com/azure/core/http/policy) and [policy in the library](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/resources/mgmt/src/main/java/com/azure/management/resources/fluentcore/policy).
```

Comment:
policy in the library  <--  maybe use full name like "policies in Azure Management Libraries"  (plural in policies?).